tests/test_scheduler.py

```diff
@@ -0,0 +1,136 @@
+"""Unit tests for scheduler module."""
+
+import unittest
+from unittest import mock
+
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import Qt, QObject,  QMimeData
+
+from iquip.apps import scheduler
+from iquip.protocols import ExperimentInfo
+
+class ExperimentModelTest(unittest.TestCase):
+    """Unit tests for ExperimentModel class."""
+
+    def setUp(self):
+        self.qapp = QApplication([])
+
+    def tearDown(self):
+        del self.qapp
+
+    def test_model_index(self):
+        data1 = []
+        for i in range(10):
+            data1.append(ExperimentInfo(str(i), {"rid": i, "priority": i}))
+        data2 = []
+        for i in range(100):
+            data1.append(ExperimentInfo(str(i), {"rid": i, "priority": 0}))
+        for data in (data1, data2):
+            mdl = scheduler.ExperimentModel()
+            mdl.experimentQueue.extend(data)
+            self.assertEqual(mdl.rowCount(), len(data))
+            for i, exp in enumerate(data):
+                self.assertEqual(mdl.data(mdl.index(i)), exp)
+
+    def test_drag_and_drop(self):
```

Comment:
If you test `dropMimeData()` method, we name it `test_drop_mime_data()`.
This is a unit test, not a functional test.